tests/test_helpers.py

```diff
@@ -0,0 +1,14 @@
+"""
+Unit-tests for helpers
+"""
+import commodore.helpers as helpers
+
+
+def test_apierror():
+    e = helpers.ApiError('test')
+    assert(f"{e}" == 'test')
+
+    try:
+        raise helpers.ApiError('test2')
+    except helpers.ApiError as e2:
+        assert(f"{e2}" == 'test2')
```

Comment:
You don't need braces for your `assert` statements. Those are not functions.

Examples:
```python
assert f"{e}" == "test"
assert f"{e2}" == "test2"
```